docs/user/operation-guide/_index.md

```diff
@@ -10,9 +10,29 @@ This guide helps you operate the Grafana Agent.
 ## Stability
 
 The core of Grafana Agent is considered stable and suitable for production use.
-Features and other functionality that are subject to change and are not
-recommended for production use will be tagged as either "beta" or
-"experimental."
+Individual features of Grafana Agent may have stability falling under one of
+the three categories:
+
+* Experimental: we are exploring a new use case and would like feedback.
```

Comment:
Small casing nits between experimental and beta

Reply:
WDYT the consistency should be?